### ISSUE TITLE
Date support for mermaid Gannt charts is missing. 

### ISSUE THREAD

**Issue by NegativeZero000** [benweet/stackedit#1359]
In the editor you could use a small sample gannt like the following. 

    ```mermaid
    gantt
    dateFormat  YYYY-MM-DD
    title Adding GANTT diagram to mermaid
    section very 
    Completed task            :done,    des1, 2014-01-06,2014-01-08
    Active task               :active,  des2, 2014-01-09, 3d
    Future task               :         des3, after des2, 5d
    Future task2              :         des4, after des3, 5d
    ```

I would expect the dates to be shown along the bottom legend but instead it is using generic week labels. w. 01, w. 02 ....

Is this status by design?

Enviroment | Value
--- | ---
Operating System | Windows 10
Browser | Chrome 68.0.3440.106

**Comment by abop** [comment]
Seems axisFormat syntax is not supported. 
Mermaid support that. 
Hope someone could drop a PR.
See [demo](https://mermaidjs.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ2FudHRcbmRhdGVGb3JtYXQgIFlZWVktTU0tRERcbmF4aXNGb3JtYXQgICVtLyVkXG50aXRsZSBTaG9w6aG555uu5Lqk5LuY6K6h5YiSXG5cbnNlY3Rpb24g6YeM56iL56KRIDAuMSBcbuaVsOaNruW6k-iuvuiuoSAgICAgICAgICA6YWN0aXZlLCAgICBwMSwgMjAxNi0wOC0xNSwgM2Rcbuivpue7huiuvuiuoSAgICAgICAgICAgIDogICAgICAgICAgIHAyLCBhZnRlciBwMSwgMmRcblxuc2VjdGlvbiDph4znqIvnopEgMC4yXG7lkI7nq6_lvIDlj5EgICAgICAgICAgICA6ICAgICAgICAgICBwMywgMjAxNi0wOC0yMiwgMjBkXG7liY3nq6_lvIDlj5EgICAgICAgICAgICA6ICAgICAgICAgICBwNCwgMjAxNi0wOC0yMiwgMTVkXG5cbnNlY3Rpb24g6YeM56iL56KRIDAuM1xu5Yqf6IO95rWL6K-VICAgICAgICAgICAgOiAgICAgICBwNiwgYWZ0ZXIgcDMsIDVkXG7kuIrnur8gICAgICAgICAgICAgICA6ICAgICAgIHA3LCBhZnRlciBwNiwgMmRcbuS6pOS7mCAgICAgICAgICAgICAgIDogICAgICAgcDgsIGFmdGVyIHA3LCAyZCIsIm1lcm1haWQiOnsidGhlbWUiOiJkZWZhdWx0In19)

**Comment by benweet** [comment]
Fixed with v5.14.10.
